jetbrains-core/src/software/aws/toolkits/jetbrains/ui/wizard/IntelliJSdkSelectionPanel.kt

```diff
@@ -54,6 +55,11 @@ class IntelliJSdkSelectionPanel(callback: AbstractNewProjectStep.AbstractCallbac
         }
     }
 
+    override fun validateAll(): List<ValidationInfo>? {
+        currentSdkPanel.validate()
```

Comment:
Took another pass at this. `null` is correct since the right thing will happen without us having to wrap any exceptions.

However, `validate()` in IntelliJ-land returns true/false and is very important for IntelliJ, since it allows validation failures without creating an error popup, so this result needs to be piped through to the IntelliJ validator as a boolean, and not a `ValidationInfo` object.

Reply:
I am confused, how is it piped though if we do nothing with the boolean result?